### ISSUE TITLE
Usability feedback (don't fail to meet WCAG, but may prove challenging for users of there service)

### ISSUE THREAD

**Issue by markrowen** [digital-land/digital-land#722]
The following section contains feedback from the audit that although do not fail to meet 
WCAG 2.2 may prove challenging for users of the service.
